Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt

```diff
@@ -200,19 +200,43 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
         }
     }
 
-    private fun getBannerSize(bannerSize: ISBannerSize): BannerAdSize? {
-        return when (bannerSize.description) {
-            "BANNER", "LARGE" -> BannerAdSize.BANNER
-            "RECTANGLE" -> BannerAdSize.VUNGLE_MREC
+    private fun getBannerSize(bannerSize: ISBannerSize): VungleAdSize? {
+        val vungleAdSize = when (bannerSize.description) {
```

Comment:
So, you set `vungleAdSize` based on `bannerSize.description` first.
Then, update `vungleAdSize` with `getAdSizeWithWidth` only when `isAdaptive` is `true`.

This logic could cover all cases, so I will follow this logic in iOS side as well.

Reply:
Yes